### ISSUE TITLE
How to save cart for specific user even after logging out?

### ISSUE THREAD

**Issue by tanaymitkari1** [MahmudulHassan5809/django-shopping-cart#1]
I actually liked this and it's quite simple to use.
Can you please help me in adding loggedin user as session id
CART_SESSION_ID = 'cart'
like even if the user logs out his cart should remain same.
Thank You

**Comment by juanpestana96** [comment]
Yes, it's really simple. You can create a Cart table, and store the cart by user. Once the user get logged in search in the table and assign the cart object to the cart session on the user.

The cart init method get the cart object from the user session with the "CART_SESSION_ID" so you can set a cart object with that session id.

`cart = self.session.get(settings.CART_SESSION_ID)`

Not tested. I will post my results

**Comment by MahmudulHassan5809** [comment]
I will try actually now very busy with my office work. But i will
definitely try it whenever i can.
Thanks for your suggestion.
